tools/darwin/packaging/darwin_embedded/mkdeb-darwin_embedded.sh.in

```diff
@@ -75,7 +75,11 @@ echo "Conflicts: $PACKAGE"                      >> $DIRNAME/$PACKAGE/DEBIAN/cont
 echo "Replaces: $PACKAGE"                       >> $DIRNAME/$PACKAGE/DEBIAN/control
 echo "Priority: Extra"                            >> $DIRNAME/$PACKAGE/DEBIAN/control
 echo "Version: $VERSION-$REVISION"                >> $DIRNAME/$PACKAGE/DEBIAN/control
-echo "Architecture: @CMAKE_SYSTEM_NAME@-@CPU@"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
+if [ "@PLATFORM@" == "iphoneos" ]; then
+  echo "Architecture: iphoneos-arm"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
+else
+  echo "Architecture: appletvos-arm64"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
```

Comment:
Maybe `@PLATFORM@-arm64`

Do we have a user with jailbroken apple tv? Would be good to know that this is correct.

Reply:
Yes we have senad.aruc on this thread https://forum.kodi.tv/showthread.php?tid=353675&page=2.
I will send him a patched tvOS deb file with `appletvos-arm64` to be sure that this is the correct value to use.

Reply:
* According to the user, with `appletvos-arm64` the install is ok with dpkg (but it seems that the user has another during the runtime).
* And with `darwin-arm64` there is an error --> `dpkg: regarding kodi-20200421-9fab130c-master-tvos_control_file_patched_darwin_arm64.deb containing org.xbmc.kodi-tvos64, pre-dependency problem:  org.xbmc.kodi-tvos64:appletvos-arm64 pre-depends on cy+cpu.arm64`

So, aside the runtime error, `appletvos-arm64` seems to be the good one (same as Memphiz yab branch)

https://forum.kodi.tv/showthread.php?tid=353675&pid=2942955#pid2942955

Reply:
Cool, so the PR is good as it is now.

Reply:
Unless you want to update the literals to `@PLATFORM@` where possible.

Reply:
Even if we use `@PLATFROM@` the if/else remains. So I vote to keep it with iphoneos and appletvos string. But if one of you want to use the variable I can change it 😉

Reply:
Up to you guys. I’m easy either way but I generally go for less string literals if possible.

Reply:
In this case I'd probably lean towards the explicit setting. Don't want to have to deal with this if something in Kodi changes. This is very specific to a single use case (jailbroken dpkg).

Reply:
👍